test/vignette/http/integration_test.clj

```diff
@@ -16,97 +19,97 @@
 (def system-local (create-system lis))
 
 (with-state-changes [(before :facts (start system-local default-port))
-                     (after :facts (stop system-local))]
+                     (after :facts (do (stop system-local)))]
```

Comment:
Not really code review, just a question. Does the `do` here that the results of `stop system-local` will be returned "up to the teardown call", rather than just executed?